app/kuma-dp/pkg/dataplane/metrics/server.go

```diff
@@ -26,11 +26,24 @@ var _ component.Component = &Hijacker{}
 
 type MetricsMutator func(in io.Reader, out io.Writer) error
 
+type QueryParametersAppender func(in *url.URL) string
+
+func EmptyQueryParametersAppender(url *url.URL) string {
```

Comment:
I would name this `ClearQueryParameters`

Reply:
Changed to `RemoveQueryParameters`